### ISSUE TITLE
[Feature Request] Option to use Postgresql or other database for storage

### ISSUE THREAD

**Issue by a0v0** [umputun/remark42#1347]
Currently BoltDb is used as database. If my server crashes all the comment/data will be lost. It is indeed possible to make backup regularly but it would be much easier to use external/managed database with remark42 and let SAAS providers worry about managing data. 

An option can be provided to use external DB or BoltDB for storage during deployment.

**Comment by umputun** [comment]
>  If my server crashes all the comment/data will be lost

This is incorrect. I"m not sure if your comment is based on some experience, but boltdb is very reliable storage, all the writes run in transactions. Our demo site runs remark42 which is "crashing" very often (sometimes multiple times a day), on each master commit as it just restarts the container, and we **never experienced any data loss**. None of user running remar42 never reported "server crashes all the comment/data will be lost" either.

Maybe you meant storing data inside your container? If this is your concern just use external volume, as the documentation suggests.

_Pls use "Issues" for the real issues/bug report. Future requests and the similar comment should go disscussions_
